src/loaders/assetLoader.js

```diff
@@ -111,18 +115,20 @@ module.exports = async function assetLoader() {
                   throw new Error(`Unknown scale ${scale} for ${filepath}`);
               }
 
+              const longname = `${`${url.replace(/\//g, '_')}_${filename}`
```

Comment:
It doesn't. It's WIP and that was moved down to help me understand usage. We can ignore this line.